python/lsst/sims/alertsim/alertsim_main.py

```diff
@@ -152,15 +160,155 @@ def iter_and_serialize(obs_data, obs_metadata, observations_field, history):
     
     """
 
-    list_of_query_dicts = []
+    list_of_alert_dicts = []
 
-    for line in obs_data.iter_catalog():
-        query_dict = dict(zip(obs_data.iter_column_names(), line))
-        list_of_query_dicts.append(query_dict)
 
-    avro_utils.catsim_to_avro(list_of_query_dicts = list_of_query_dicts)
+    if not history:
+        for line in obs_data.iter_catalog():
+            diaSource_dict = dict(zip(obs_data.iter_column_names(), line))
+            alert_dict = {'alertId':45135, 'l1dbId':12545, 
+                'diaSource':diaSource_dict}
+            list_of_alert_dicts.append(alert_dict)
+    else:
+        from lsst.sims.catUtils.utils import FastStellarLightCurveGenerator
+        lc_gen = FastStellarLightCurveGenerator(obs_data.db_obj, opsim_path)
+        #from lsst.sims.catUtils.utils import StellarLightCurveGenerator
+        #lc_gen = StellarLightCurveGenerator(obs_data.db_obj, opsim_path)
+        
+        print("#### radius = %f" % (radius))
+        ra1=obs_metadata.pointingRA - radius
```

Comment:
You will need to make this
```
ra1=obs_metadata.pointingRA - radius*np.cos(np.radians(obs_metadata.pointingDec))
ra2=obs_metadata.pointingDec + radius*np.cos(np.radians(obs_metadata.pointingDec))
```
to account for the fact that a change of 1 degree in RA means a different absolute angular distance at the poles than at the equator

Reply:
Note the edit above.  You need to divide by cos(dec), not multiply

I guess that implies problems at the poles.  I'm not sure how we should deal with them.

Reply:
ok